jenkins-ci/Jenkinsfile

```diff
@@ -123,10 +120,21 @@ pipeline {
                 withEnv(["JAVA_HOME=${ tool 'jdk8' }", "PATH+MAVEN=${tool 'maven-3.5.2'}/bin:${env.JAVA_HOME}/bin"]) {
                     sh '''
                         export USE_COLLECTOR_OR_AGENT=collector
-                        rm ./jaeger-java-test -rf
-                        git clone https://github.com/Hawkular-QE/jaeger-java-test.git
                         cd jaeger-java-test
-                        export JAEGER_FLUSH_INTERVAL=2500
+                        mvn clean test
+                    '''
+                }
+            }
+        }
+        stage('production-elastic: functional test with edge route'){
+            steps{
+                withEnv(["JAVA_HOME=${ tool 'jdk8' }", "PATH+MAVEN=${tool 'maven-3.5.2'}/bin:${env.JAVA_HOME}/bin"]) {
+                    sh '''
+                        export USE_COLLECTOR_OR_AGENT=collector
```

Comment:
@kevinearls what this test actually does? How many spans it reports?

Is ES cleaned before running this?

Reply:
@pavolloffay ES is not clean before running this, but the tests are validated by retrieving spans created since the test started.

This is really just a smoke test.  The tests run can be found here https://github.com/Hawkular-QE/jaeger-java-test.git and only create 12-15 spans in total.  This can certainly be expanded if necessary, but the initial idea was that this would just be a smoke test, not a full functional test.

Reply:
The smoke test is what we want. I was afraid that it creates more than is necessary.